tests/integration/transactions/test_mptoken_authorize.py

```diff
@@ -0,0 +1,114 @@
+# TODO: uncomment test module when rippled Docker image supports MPT feature
+# from tests.integration.integration_test_case import IntegrationTestCase
+# from tests.integration.it_utils import (
+#     fund_wallet_async,
+#     sign_and_reliable_submission_async,
+#     test_async_and_sync,
```

Comment:
MPT is now in devnet, so there should be a docker image available now to uncomment these